Formula/httping.rb

```diff
@@ -19,18 +17,16 @@ class Httping < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "0090bfe517e71dd18d61ed96c5f5cf2cc1bf252399c9d49b521b9a53bc7a46b8"
   end
 
-  disable! date: "2022-10-19", because: :repo_removed
-
   depends_on "gettext"
   depends_on "openssl@1.1"
-
   uses_from_macos "ncurses"
 
   def install
+    # Reported upstream, see: https://github.com/folkertvanheusden/HTTPing/issues/4
+    inreplace "utils.h", "useconds_t", "unsigned int"
     # Reported upstream, see: https://github.com/Homebrew/homebrew/pull/28653
-    inreplace %w[configure Makefile], "ncursesw", "ncurses"
+    inreplace %w[configure Makefile], "lncursesw", "lncurses"
```

Comment:
Could you report this upstream too?

Reply:
https://github.com/folkertvanheusden/HTTPing/issues/7

Reply:
Could you replace the commented link above with this link then